h/views/api/groups.py

```diff
@@ -80,7 +81,7 @@ def create(request):
     link_name="group.read",
     description="Fetch a group",
 )
-def read(context, request):
+def read(context: GroupContext, request):
```

Comment:
At some point when writing the tests (and writing test code to pass autospec'd mocks of `GroupContext` into the views) I got confused by some outdated comments and docstrings and thought that the different views accepted different types of `context` object (`GroupContext`, `GroupUpsertContext`, ...). Actually all the views in this file use the same `GroupContext` type. Adding it as type annotations to clarify this. This'll also help with code completion, linting, type-checking.